yal/askaquestion.py

```diff
@@ -165,14 +163,20 @@ async def state_display_results(self):
                     "answer, reply with the number of the topic you're interested "
                     "in.",
                     "",
-                    get_display_choices(choices),
+                    get_display_choices(choices, bold_numbers=True),
+                    "",
+                    "or",
+                    f"*{len(answers)+1}*. Back to first list",
+                    f"*{len(answers)+2}*. Talk to a counsellor",
```

Comment:
Shouldn't this rather be `len(choices)`? I think it might be the same at this point, but I think it would make more sense for this to match what is being passed to `get_display_choices`

Reply:
Done.